### PR TITLE
Correct release date on ComputationalWorkflow and FormalParameter

### PR DESCRIPTION
**Pull request by AlasdairGray**
Update the published release data on the ComputationalWorkflow profile and type, and FormalParameter profile to reflect the changes made this year.

### CODE REVIEW

**Review by alaninmcr — COMMENTED** [review]
Are the changes the wrong way round? It looks as if it has been changed from 2021 to 2020.

**Comment by AlasdairGray** [comment]
Don't think so. New dates are 4 and 9 March respectively (based on Git commits).

The dates displayed with 2021 dates when I ran things locally.

**Comment by alaninmcr** [comment]
I approve

File: _types/ComputationalWorkflow/1.0-RELEASE.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 previous_version: "0.1-DRAFT-2020_07_21"
 previous_release:
 
-dateModified: 2020-10-19
+dateModified: 2021-03-04
```
Comment (ljgarcia) [inline _types/ComputationalWorkflow/1.0-RELEASE.html:17]: Should it not be 2021-03-09 as in the version date 20210309T210904 ?

Comment (AlasdairGray) [reply _types/ComputationalWorkflow/1.0-RELEASE.html:17]: The last edit on the type was 4 March I think.

Comment (ljgarcia) [reply _types/ComputationalWorkflow/1.0-RELEASE.html:17]: Ok

**Review by ljgarcia — APPROVED** [review]
Looks good

**Pull request merged**